### PR TITLE
Refinements

### PR DESCRIPTION
**Pull request by naszam**

### CODE REVIEW

File: spell/spell-reviewer-goerli-checklist.md [hunk 1]
```diff
@@ -2,10 +2,6 @@
 
 ## Goerli
 
-PR: https://github.com/makerdao/spells-goerli/pull/TODO
-
-Exec date: YYYY-MM-DD
```
Comment (The-Arbiter) [inline spell/spell-reviewer-goerli-checklist.md:7]: If we are removing this let's make sure that there is a check somewhere else for the date being correct, as this functioned as a defacto check

File: spell/spell-reviewer-goerli-checklist.md [hunk 2]
```diff
@@ -96,6 +93,7 @@ Spell Actions:
     * [ ] `tau` parameter used is the old `tau` value
   * [ ] Autoline (`line`) + Liquidation Oracle Price Bump (`val`)
     * [ ] Enable Autoline
+      * [ ] `ilk` follows format "RWAXXX-A"
```
Comment (The-Arbiter) [inline spell/spell-reviewer-goerli-checklist.md:96]: Not the case for D3M fyi

File: spell/spell-reviewer-goerli-checklist.md [hunk 3]
```diff
@@ -121,10 +119,11 @@ Spell Actions:
     * [ ] Cliff Duration matches Doc (`eta`)
     * [ ] Restricted (by default)
     * [ ] Manager match Doc (`mgr`, set to zero for DAI streams by default)
+    * [ ] IF `DssVestTransferrable` is used
+      * [ ] Ensure `DssVestTransferrable` Allowance matches total transfer amount
```
Comment (The-Arbiter) [inline spell/spell-reviewer-goerli-checklist.md:123]: I kept getting stuck on this, can you please elaborate how I would go about checking this? Is there a contract you're reading a value off?

**Review by The-Arbiter — COMMENTED** [review]
Small requests

File: spell/spell-reviewer-goerli-checklist.md (hunk 1 above)
Comment (naszam) [reply spell/spell-reviewer-goerli-checklist.md:7]: done in 67a622e, to check the spell `description` date when generating the exec hash

Comment (naszam) [reply spell/spell-reviewer-goerli-checklist.md:7]: included in the crafter checklist as well in 788043c

File: spell/spell-reviewer-goerli-checklist.md (hunk 2 above)
Comment (naszam) [reply spell/spell-reviewer-goerli-checklist.md:96]: what you mean? It should refer to RWA only

File: spell/spell-reviewer-goerli-checklist.md (hunk 3 above)
Comment (naszam) [reply spell/spell-reviewer-goerli-checklist.md:123]: yeah, this could be rephrased better

Comment (naszam) [reply spell/spell-reviewer-goerli-checklist.md]: done in 9e2dcf5 (see for references [spell archive](https://github.com/makerdao/spells-mainnet/blob/master/archive/2022-10-05-DssSpell/DssSpell.sol#L88) and [vest contract](https://github.com/makerdao/dss-vest/blob/master/src/DssVest.sol#L469)).

File: spell/spell-reviewer-goerli-checklist.md [hunk 4]
```diff
@@ -66,6 +62,7 @@ Spell Actions:
 * [ ] Onboarding
   * [ ] [Collateral Onboarding](./collateral-onboarding-checklist.md)
   * [ ] [RWA Onboarding](./rwa-onboarding-checklist.md)
+  * [ ] [Teleport Onboarding](./teleport-oboarding-checklist.md)
```
Comment (The-Arbiter) [inline spell/spell-reviewer-goerli-checklist.md:65]: Typo here

```suggestion
  * [ ] [Teleport Onboarding](./teleport-onboarding-checklist.md)
```

File: spell/spell-reviewer-mainnet-checklist.md [hunk 5]
```diff
@@ -19,6 +15,7 @@ Spell Actions:
   * [ ] `community` commit is the latest change or merge commit
   * [ ] `community` repo url matches
   * [ ] script matches
+  * [ ] `description` date in `DssSpell.sol` matches exec copy one
```
Comment (The-Arbiter) [inline spell/spell-reviewer-mainnet-checklist.md:18]:
```suggestion
  * [ ] `description` date in `DssSpell.sol` matches exec copy
```
Can drop the 'one' here if you want

File: spell/spell-reviewer-mainnet-checklist.md [hunk 6]
```diff
@@ -70,6 +67,7 @@ Spell Actions:
 * [ ] Onboarding
   * [ ] [Collateral Onboarding](./collateral-onboarding-checklist.md)
   * [ ] [RWA Onboarding](./rwa-onboarding-checklist.md)
+  * [ ] [Teleport Onboarding](./teleport-oboarding-checklist.md)
```
Comment (The-Arbiter) [inline spell/spell-reviewer-mainnet-checklist.md:70]: Typo again

File: spell/spell-reviewer-mainnet-checklist.md [hunk 7]
```diff
@@ -126,10 +124,11 @@ Spell Actions:
     * [ ] Cliff Duration matches Doc (`eta`)
     * [ ] Restricted (by default)
     * [ ] Manager match Doc (`mgr`, set to zero for DAI streams by default)
+    * [ ] IF [DssVestTransferrable](https://github.com/makerdao/dss-vest/blob/master/src/DssVest.sol#L463) is used
```
Comment (The-Arbiter) [inline spell/spell-reviewer-mainnet-checklist.md:127]: Transferable has 1 r but it's already deployed 😂 I guess it's ambiguous but doesn't matter

File: spell/spell-reviewer-mainnet-checklist.md [hunk 8]
```diff
@@ -152,9 +151,11 @@ Spell Actions:
   * [ ] Optimization Enabled: No
   * [ ] Other Settings: default evmVersion, GNU AGPLv3 license
 * [ ] Deployed Spell Code matches GitHub
-  * diffcheck etherscan source against spell PR (i.e. via vscode `code --diff etherscan.sol github.sol`)
+  * [ ] diffcheck etherscan source against spell PR (i.e. via vscode `code --diff etherscan.sol github.sol`)
```
Comment (The-Arbiter) [inline spell/spell-reviewer-mainnet-checklist.md:154]: How do you get etherscan.sol here? Are you copy pasting manually?

**Review by The-Arbiter — APPROVED** [review]

File: spell/spell-reviewer-mainnet-checklist.md (hunk 7 above)
Comment (naszam) [reply spell/spell-reviewer-mainnet-checklist.md:127]: I think it was intended and it seems it's also accepted as variant

File: spell/spell-reviewer-mainnet-checklist.md (hunk 6 above)
Comment (naszam) [reply spell/spell-reviewer-mainnet-checklist.md:70]: hmm, interesting, good catch! It seems that the CI didn't catch it

Comment (naszam) [reply spell/spell-reviewer-mainnet-checklist.md]: done in c0b66d1

**Review by The-Arbiter — APPROVED** [review]
LGTM

File: spell/spell-reviewer-goerli-checklist.md [hunk 9]
```diff
@@ -9,11 +9,12 @@ Spell Actions:
...
 ## Checklist
 * [ ] Office Hours
-  * [ ] off (default, spell casted after deploying, pause delay 60 sec.)
+  * [ ] Off (default, spell casted after deploying, pause delay 60 sec.)
 * [ ] Exec Hash
   * [ ] empty (default)
 * [ ] 30 Days Expiry
-  * [ ] weekly (monthly discarded)
+  * [ ] weekly
+  * [ ] monthly (discarded)
```
Comment (The-Arbiter) [inline spell/spell-reviewer-goerli-checklist.md:17]: Not much context here as to what discarded means.. 

If we don't use monthly, we shouldn't have it as an option
If you mean 'discard' weekly in favour of monthly, we don't need to have it in brackets here

**Review by The-Arbiter — APPROVED** [review]

File: spell/spell-reviewer-goerli-checklist.md (hunk 9 above)
Comment (naszam) [reply spell/spell-reviewer-goerli-checklist.md:17]: yeah, we should probably remove it.

Comment (naszam) [reply spell/spell-reviewer-goerli-checklist.md]: done in 1765987, where I removed both `weekly` and `monthly` as now the labels have been removed in `config.sol` (see [old spells](https://github.com/makerdao/spells-mainnet/blob/master/archive/2021-05-14-DssSpell/DssSpell.t.sol#L137) in archive for reference for labels and expiration)

**Review by The-Arbiter — APPROVED** [review]
LGTM

**Pull request merged**